app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInActivity.kt

```diff
@@ -61,6 +63,7 @@ class LoggedInActivity : BaseActivity(R.layout.activity_logged_in) {
 
     private val welcomeViewModel: WelcomeViewModel by viewModel()
     private val loggedInViewModel: LoggedInViewModel by viewModel()
+    private val hAnalytics: HAnalytics by inject()
```

Comment:
Doesn't seem to be used, maybe added by accident and then realized it wasn't needed?